### ISSUE TITLE
Can't add an integration for #prjbureau

### ISSUE THREAD

**Issue by whitequark** [matrix-org/libera-chat#18]
I clicked this button:
![Screenshot_20230703_195346](https://github.com/matrix-org/libera-chat/assets/54771/14a40bcd-690f-4cd3-93c4-07e7f826096e)

It did not send me a link. Now it's showing me an error when I try to do that again:
![Screenshot_20230703_195415](https://github.com/matrix-org/libera-chat/assets/54771/3f9fe0af-2b42-47e4-8686-21d27c8fbf0f)

**Comment by whitequark** [comment]
Could one of the bridge operators bridge my rooms (I can provide a list) manually, please? This is an extremely disruptive change at a fairly short notice and as I came to understand, the Libera folks are not able to operate the bridge at all.

**Comment by whitequark** [comment]
No longer relevant since I chose to run my own bridging infrastructure and not depend on issues no one seems to look at.
